src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,147 @@
+| [stagehand](https://pub.dartlang.org/packages/stagehand) | A Dart project generator. WebStorm and IntelliJ use Stagehand templates when you create a new application, but you can also use the templates from the command line via [pub global activate](https://www.dartlang.org/tools/pub/cmd/pub-global). | Templates include console-full, console-simple, server-shelf, web-angular, and web-simple. |
+| [test](https://pub.dartlang.org/packages/test) | A standard way of writing and running tests in Dart. | expect(), group(), test() |
+{:.table .table-striped .nowrap}
+
+To find more packages, see [pub.dartlang.org](https://pub.dartlang.org/).
+
+## Packages that correspond to SDK libraries
+
+These "expansion pack" libraries have the same name as their counterpart in the SDK.
+Each of these libraries builds upon an SDK library, adding
+additional functionality and filling in missing features:
+
+| **Package** | **Description** | **Commonly used APIs** |
+| [async](https://www.dartdocs.org/documentation/async/latest/) | Utility classes to work with asynchronous computations. For more information, see [Unboxing Packages: async part 1](http://news.dartlang.org/2016/03/unboxing-packages-async-part-1.html), [part 2](http://news.dartlang.org/2016/03/unboxing-packages-async-part-2.html), and [part 3.](http://news.dartlang.org/2016/04/unboxing-packages-async-part-3.html) | AsyncMemoizer, CancelableOperation, FutureGroup, LazyStream, Result, StreamCompleter, StreamGroup, StreamSplitter |
+| [collection](https://www.dartdocs.org/documentation/collection/latest) | Utility functions and classes to make working with collections easier. For more information, see [Unboxing Packages: collection.](http://news.dartlang.org/2016/01/unboxing-packages-collection.html) | Equality, CanonicalizedMap, MapKeySet, MapValueSet, PriorityQueue, QueueList |
+|[convert](https://www.dartdocs.org/documentation/convert/latest/) | Contains encoders and decoders for converting between different data representations. Note that one of the data representations is commonly known as URL-encoded (named `percent` here). | HexDecoder, PercentDecoder |
+{:.table .table-striped .nowrap}
+
+## Specialized libraries
+
+This page doesn't include some of the more specialized libraries that are covered
+elsewhere.
+
+### Web libraries
+
+If you write web apps, check out Angular 2, a web application framework.
+Other available resources are the
+[js](https://www.dartdocs.org/documentation/js/latest) package
+for interoperability with JavaScript APIs, and the
+[dart:html](https://api.dartlang.org/stable/dart-html/dart-html-library.html)
+library for low-level HTML programming.<br>
+More information: [webdev.dartlang.org]({{site.webdev}})
+
+### Flutter libraries
+
+Use Flutter to write mobile applications.
```

Comment:
The first sentence doesn't fit in with the other ones. It doesn't describe the libraries, and it sounds bossier than we need to be here. Maybe:

If you write mobile apps, check out Flutter.